kevm-pyk/src/kevm_pyk/cli.py

```diff
@@ -339,6 +339,19 @@ def explore_args(self) -> ArgumentParser:
             default='kore-rpc',
             help='Custom command to start RPC server',
         )
+        args.add_argument(
+            '--failure-information',
+            dest='failure_info',
+            default=True,
+            action='store_true',
+            help='Show failure summary for all failing tests',
+        )
+        args.add_argument(
+            '--no-failure-information',
+            dest='failure_info',
+            action='store_false',
+            help='Do not show failure summary for failing tests',
+        )
```

Comment:
Do you know if both arguments are required? Is there a reason not to have only `--no_failure-information` with `default=False`?

Reply:
If the default is `True`, you have to supply both options so that it's possible to default to `True` but still set to `False` as needed. Here we want the default to be `True`..

Reply:
It is not strictly required. I was told we usually add both positive and negative flags when the default is is true, for consistency, but not when the default is false. Since there's two different versions of the failure info flag, one with default false and the other with default true, I just included the positive and negative options so the two flag variants behave consistently apart from the default value.